pom.xml

```diff
@@ -54,6 +70,48 @@
 			<artifactId>mailer</artifactId>
 			<version>1.5</version>
 		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins.workflow</groupId>
+			<artifactId>workflow-step-api</artifactId>
+			<version>${workflow.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>matrix-project</artifactId>
```

Comment:
I don't think so, it's a hard dependency (imports from `matrix-project` in the code mainly to skip lock in defined cases).